src/binary-search-tree.js

```diff
@@ -31,8 +64,18 @@ class BinarySearchTree {
   // You'll need the queue-helper file for this. Or could you roll your own queue
   // again. Whatever floats your boat.
   breadthFirstForEach(cb) {
-
-  }
+    const current = [node];
+    
+        while (current.length > 0) {
+          const next = [];
+          for (const node of current) {
+            cb(node);
+            if (node.left) next.push(node.left);
+            if (node.right) next.push(node.right);
+          }
+          current = next;
```

Comment:
You won't be able to assign to `current` here since you declared `current` using `const`.